dev/release/setup-rhel-rebuilds.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh -l
```

Comment:
Filename is wrong: `setup-rhel-rebuilds.sh` -> `setup-rhel-clone.sh`

Reply:
RHEL does not like the use of clone, as an example Wikpedia uses derivatives https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives

Reply:
See also https://wiki.centos.org/FAQ/General#What_is_CentOS.27s_relationship_with_Red_Hat.2BAK4.2C_Inc._or_RHEL.3F where it is stated clone is inappropriate